Changes implements the Trusted Extension functionality in universal filter
which calls labeltops filter to emit postscript banner and trailer pages.

This patch together with tx.patch in cups and libcupsfilters makes the
Trusted Extension implementation complete.

--- a/mime/cupsfilters-universal.convs
+++ b/mime/cupsfilters-universal.convs
@@ -41,6 +41,7 @@
 text/plain                      application/vnd.universal-input      0    -
 application/PCLm                application/vnd.universal-input      0    -
 application/vnd.cups-pdf-banner application/vnd.universal-input      0    -
+application/vnd.cups-labeled-banner application/vnd.universal-input  0    -
 
 ########################################################################
 #
--- a/mime/cupsfilters.types
+++ b/mime/cupsfilters.types
@@ -99,6 +99,7 @@ #
 
 application/vnd.cups-pdf
 application/vnd.cups-pdf-banner     string(0,'#PDF-BANNER')
+application/vnd.cups-labeled-banner     string(0,'#LABELED-BANNER')
 application/PCLm
 
 ########################################################################
